Thrall mission (#6441)
endless-sky/endless-sky: data/human/human missions.txt

			label payup
			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
				accept
	
mission "Thrall Emigration"

> 	name "Thrall Emigration"
> 	description "Transport a family from Thrall to <destination>, where they hope to find better employment. They cannot pay you."

Reviewer: ```suggestion
	name "<origin> Emigration"
	description "Transport a family from <origin> to <destination>, where they hope to find better employment. They cannot pay you."
```

	minor
	source
		planet "Thrall"
	destination "Delve"
	to offer
		random < 50
	passengers 2
	on offer
		conversation

> 			`By the edge of the spaceport at night, you see a thin sweaty man turn away from a pilot, disheartened. You make eye contact, and he begins hurrying over to you.`

Reviewer: ```suggestion
			`By the edge of the spaceport at night, you see a thin, sweaty man turn away from a pilot, disheartened. You make eye contact, and he begins hurrying over to you.`
```

			choice
				`	(Wait for him.)`
				`	(Walk away quickly.)`
					decline

> 			`	"Excuse me! Are you a pilot?" Without waiting for a response, he continues "I need to go to the Syndicate. I work all day picking cotton with these hands, but my family's still starving. We can't afford transport..."`

Reviewer: ```suggestion
			`	"Excuse me! Are you a pilot?" Without waiting for a response, he continues, "I need to go to the Syndicate. I work all day picking cotton with these hands, but my family's still starving. We can't afford transport..."`
```

> 			`	You can barely understand him through his accent. The man's teeth look worse than his clothes, and his breath makes you step back. His eyes are human.`

Reviewer: What was your intent with including `His eyes are human`? As-is, it comes off more like a strange confirmation that this person isn't an alien. I think this could maybe use some rewording if this wasn't your intended meaning.

Author: ```suggestion
			`	You can barely understand him through his accent. The man's teeth look worse than his clothes, and his breath makes you step back. His eyes implore you.`
```
The original intent was to establish commonality after the previous description, but I was thinking in terms of empathy rather than species. The wording is kind of awkward as well: would something like this work better? 
Thanks for all the comments. I committed your other suggestions.

Reviewer: That works great!

			choice
				`	"I might be able to help you. What planet do you need to reach?"`
				`	"I'm sorry, I only transport for pay."`
					goto disappointment

> 			`	His eyes brighten with hope. "Oh! I, I heard some people going to Delve to be miners. They hire folk like us. I, if you could get me, and my wife and our two sons there, I'd praise you to the high stars. You'd save our lives."`

Author: Good catch, accepted.

			choice
				`	"I can take you, but no more."`
					goto separation
				`	"I'd be happy to offer you four transport."`
					goto offer
				`	"Sorry, I think this won't work out."`
					goto disappointment
			label separation
			`	"The two little ones are really small," he persists. "We can cramp together. You'll just notice two of us. I mean, some go themselves and send money back, but..."`
			`	He shakes his head. "My wife's already told me we need to stay together, and I agree. I can't leave. Please, nobody else has said yes."`
			choice
				`	"I guess I can find room."`
					goto offer
				`	"Sorry, but that's still too many. Have a nice day."`
					goto disappointment
			label offer

> 			`	"Oh, thank you, thank you!" he says, vigorously shaking your hand. "I'll get them fast, we can leave tonight! I'm Gaint, by the way" He quickly disappears, turning back at least three times to tip his hat to you and make sure you haven't left.`

Reviewer: ```suggestion
			`	"Oh, thank you, thank you!" he says, vigorously shaking your hand. "I'll get them fast, we can leave tonight! I'm Gaint, by the way." He quickly leaves to get them, turning back at least three times to tip his hat to you and make sure you haven't left.`
```
He doesn't exactly `disappear` if he turns back to tip his hat several times, no?

> 			`	When Gaint's family arrives, your heart warms. His wife, Mell, is carrying their infant child, and a toddler clutches on her leg. His fingers look like a skeleton, and his eyes look like his dad's. "Thank you so much," Mell says as you load them onto the ship.`

Reviewer 2: The phrase 'your heart warms' is kind of corny, and moreover dialogs aren't supposed to tell the player how they feel.

Reviewer 3: I like it when it suggest emotions but maybe it should not be so affirmative of what you feel

Author: ```suggestion
			`	Gaint's family arrives quickly. His wife, Mell, is carrying their infant child, and a toddler clutches her leg. His fingers look like a skeleton, and his eyes look like his dad's. "Thank you so much," Mell says as they enter your ship.`
```
Agreed, thanks for pointing that out. Changed a few other words as well. How does this version read?

Reviewer 2: That's a lot better, IMO.

				accept
			label disappointment
			`	You fend off a few more pleas before he understands it's hopeless and begins looking for someone else.`
				decline
	
	on visit
		dialog phrase "generic passenger on visit"
	
	on complete
		conversation

> 			`	As your ship enters Delve's cavernous spaceport, you can sense a mixture of hope and apprehension in Gaint and Mell. "Andrin talks too much. I didn't know it's all underground," you catch Gaint saying. Mell responds "We'll figure it out." When they notice you, they both become very thankful once again. They collect their things and quickly leave the ship. You wish them well on their way out.`

Reviewer 4: ```suggestion
			`	As your ship enters Delve's cavernous spaceport, you can sense a mixture of hope and apprehension in Gaint and Mell. Andrin talks too much. "I didn't know it's all underground," you catch Gaint saying. Mell responds "We'll figure it out." When they notice you, they both become very thankful once again. They collect their things and quickly leave the ship. You wish them well on their way out.`
```

Seems this quote might have been misplaced?

Author: ```suggestion
			`	As your ship enters Delve's cavernous spaceport, you can sense a mixture of hope and apprehension in Gaint and Mell. "Andrin talks too much," you overhear Gaint saying. "I didn't know it's all underground."' 
			`	"We'll figure it out," Mell assures him. When they notice you, the conversation stops. Your ship lands, and the family collects their things to leave. They thank you again on their way out.`
```
The full quote is the intent, but it could definitely be clearer. How does this version look? I also changed the ending to be in line with the other suggestion thread.

Reviewer: ```suggestion
			`	As your ship enters Delve's cavernous spaceport, you can sense a mixture of hope and apprehension in Gaint and Mell. "Andrin talks too much," you overhear Gaint saying. "I didn't know it's all underground."`
			`	"We'll figure it out," Mell assures him. When they notice you're listening in, the conversation stops. Your ship lands, and the family collects their things to leave. They thank you again on their way out.`
```
Just used your base and added/corrected a bit myself. How's this look?

Author: Sounds good, committed.

> 		

Reviewer 2: I'm not super familiar with mission syntax, but I'm pretty sure that there's supposed to be a `decline` here. If anyone more informed about how missions work could share their input, I'd be grateful.

Reviewer 3: there is always an implicit decline at the end of each conversation
also there is no need for a decline when the conversation is in the on complete, decline makes sense for on offer, to decline the mission or anywhere else just to end the conversation at that place

			label payup

> 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
> 				accept
> 	
>   
> 
> mission "Thrall Emigration"
> 	name "<origin> Emigration"

Reviewer: ```suggestion
			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
				accept

mission "Thrall Emigration"
	name "<origin> Emigration"
```

mission "Thrall Emigration"
	name "<origin> Emigration"
	description "Transport a family from <origin> to <destination>, where they hope to find better employment. They cannot pay you."
	minor

> 	source
> 		planet "Thrall"

Reviewer: ```suggestion
	source "Thrall"
```

	destination "Delve"
	to offer
		random < 50
	passengers 2
	on offer
		conversation
			`By the edge of the spaceport at night, you see a thin, sweaty man turn away from a pilot, disheartened. You make eye contact, and he begins hurrying over to you.`
			choice
				`	(Wait for him.)`
				`	(Walk away quickly.)`
					decline
			`	"Excuse me! Are you a pilot?" Without waiting for a response, he continues, "I need to go to the Syndicate. I work all day picking cotton with these hands, but my family's still starving. We can't afford transport..."`
			`	You can barely understand him through his accent. The man's teeth look worse than his clothes, and his breath makes you step back. His eyes implore you.`
			choice
				`	"I might be able to help you. What planet do you need to reach?"`
				`	"I'm sorry, I only transport for pay."`
					goto disappointment

> 			`	His eyes brighten with hope. "Oh! I, I heard some people are going to Delve to be miners. They hire folk like us. I, if you could get me, and my wife and our two sons there, I'd praise you to the high stars. You'd save our lives."`

Reviewer: ```suggestion
			`	His eyes brighten with hope. "Oh! I, I heard some people are going to <planet> to be miners. They hire folk like us. I, if you could get me, and my wife and our two sons there, I'd praise you to the high stars. You'd save our lives."`
```

			`	He shakes his head. "My wife's already told me we need to stay together, and I agree. I can't leave. Please, nobody else has said yes."`
			choice
				`	"I guess I can find room."`
					goto offer
				`	"Sorry, but that's still too many. Have a nice day."`
					goto disappointment
			label offer
			`	"Oh, thank you, thank you!" he says, vigorously shaking your hand. "I'll get them fast, we can leave tonight! I'm Gaint, by the way." He quickly leaves to get them, turning back at least three times to tip his hat to you and make sure you haven't left.`
			`	Gaint's family arrives quickly. His wife, Mell, is carrying their infant child, and a toddler clutches her leg. His fingers look like a skeleton, and his eyes look like his dad's. "Thank you so much," Mell says as they enter your ship.`
				accept
			label disappointment
			`	You fend off a few more pleas before he understands it's hopeless and begins looking for someone else.`
				decline
	
	on visit
		dialog phrase "generic passenger on visit"
	
	on complete
		conversation

> 			`	As your ship enters Delve's cavernous spaceport, you can sense a mixture of hope and apprehension in Gaint and Mell. "Andrin talks too much," you overhear Gaint saying. "I didn't know it's all underground."`

Reviewer: ```suggestion
			`	As your ship enters <planet>'s cavernous spaceport, you can sense a mixture of hope and apprehension in Gaint and Mell. "Andrin talks too much," you overhear Gaint saying. "I didn't know it's all underground."`
```
